pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -84,18 +86,20 @@ func (c *APIReconciler) reconcile(ctx context.Context, apiExport *apisv1alpha1.A
 		identities[gr] = apiExport.Status.IdentityHash
 	}
 
+	clusterName := logicalcluster.From(apiExport)
+
 	// Find schemas for claimed resources
 	claims := map[schema.GroupResource]*apisv1alpha1.PermissionClaim{}
 	for i := range apiExport.Spec.PermissionClaims {
 		pc := &apiExport.Spec.PermissionClaims[i]
```

Comment:
Add `pc` to the logger here

Reply:
Done

Reply:
You can still add the field early and keep it around for others to use ;)